### ISSUE TITLE
How can we collect test metadata for CircleCI?

### ISSUE THREAD

**Issue by matthewconstantine** [emberjs/ember-mocha#135]
We'd like to get metadata about the tests we run. [CircleCI recommends](https://circleci.com/docs/test-metadata/#a-namemochajsamocha-for-nodejs) using mocha-junit-reporter. How might we configure ember-mocha to use it?

**Comment by Turbo87** [comment]
I don't think this is currently possible since we use testem between the tests in the browser and the console output. It looks like the reporter you mentioned is only relevant for node.js tests.
